src/containers/CreatePipelineRun/YAMLEditor.test.js

```diff
@@ -211,4 +212,17 @@ describe('YAMLEditor', () => {
       expect(getByText(/Error creating PipelineRun/)).toBeTruthy();
     });
   });
+
+  it('during loading loading message should be displayed', () => {
+    const { getAllByText } = renderWithRouter(
+      <CreateYAMLEditor loading loadingMessage="wait. test is in progress" />
+    );
+    expect(getAllByText(/wait. test is in progress/)).toBeTruthy();
+  });
+
+  it('during loading button create should be disabled and button cancel should not', () => {
+    const { queryAllByText } = renderWithRouter(<CreateYAMLEditor loading />);
+    expect(submitButton(queryAllByText).disabled).toBe(true);
+    expect(cancelButton(queryAllByText).disabled).toBe(false);
+  });
```

Comment:
these two tests could be combined as they're part of the same piece of functionality, one test for the loading state should be sufficient as it's not particularly complex